htop.c

```diff
@@ -189,6 +193,10 @@ static CommandLineSettings parseArguments(int argc, char** argv) {
 
             break;
          }
+         case 'I': {
+            flags.highlightNewOldNsec = optarg ? atoi(optarg) : 0;
```

Comment:
This will set highlightNewOldNsec = -1 on invalid numeric input (e.g. to large numbers, or negative numbers).
Some basic error handling as in the -d (--delay) case would be nice.